### PR TITLE
changes tags for academia to academic and add talks tags to google talk

### LINKED ISSUE #97: Update post tagging by amalgamating tags, ensure proper tagging
You may not have noticed, but we have both an "academic" and "academia" tag! Furthermore, a number of our events aren't fully tagged. It would be cool to go through the tag list and amalgamate some of these duplicate tags, as well as drudging through all of the articles and ensuring they have a full list of tags.

You can find the tag page [here](http://wics.uwaterloo.ca/tags/).

### PR DESCRIPTION
**Pull request by jyuwono**
Resolves #97

### CODE REVIEW

File: content/S2015/S15-event-googletalk.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Title: Google Talk
 Date: 2015-05-14 18:30
 Category: Events
-Slug: google-talk
+Slug: google-talk, talks
```
Comment (fboxwala) [inline content/S2015/S15-event-googletalk.md:4]: So this line is actually for a Slug, not tags. There should only be one slug (it's what will come up in the URL after wics.uwaterloo.ca/ but you might know that already :P). You need to add a new line for Tags looks like this file doesnt have any

**Comment by fboxwala** [comment]
So I just said this on Andee's PR as well, we only like to have one file per PR. Could you make the change above and split into two PR's?

**Comment by ehashman** [comment]
Since this is addressing #97, I changed your description to reflect that. When you use the phrasing "Fix #97" or "Resolve #97", GitHub will automatically close the issue when merged.

However, this commit only addresses two of our files but the issue affects many more, so I'm curious if you are interested in fixing the entire issue or just a small part.

Ideally, if one were to fix the issue, there would be two commits: one in which all the tags are amalgamated and one in which tags are added to old articles. I think we are missing quite a few; someone needs to go through every article in the backlog and the tag list and ensure they all have the relevant tags.

File: content/W2015/W15-event-research-showcase.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Title: Research Showcase and Mixer
 Date: 2015-02-04 17:30
 Category: Events
-Tags: talks, academia, social
+Tags: talks, academic, social
```
Comment (ehashman) [inline content/W2015/W15-event-research-showcase.md:4]: Do we want to do academia -> academic, or academic -> academia? I'm not sure which tag sounds better.

File: content/S2015/S15-event-googletalk.md (hunk 1 above)
Comment (jyuwono) [inline content/S2015/S15-event-googletalk.md:4]: oh thanks for pointing that out! Sorry, I didn't realize that before.

File: content/W2015/W15-event-research-showcase.md (hunk 2 above)
Comment (jyuwono) [inline content/W2015/W15-event-research-showcase.md:4]: 1. I checked every tags in every pages, and there is only 1 "academia" and multiple "academic"(s). So, I was assuming that we actually want to use academic. However, if you want it to be academia, then I don't mind changing the others.
2. I can go through everything again this week and add all the missing tags, but I'm not too sure what you mean by 2 commits.
   Thank you for the comments!

Comment (ehashman) [inline content/W2015/W15-event-research-showcase.md:4]: 1. I am suggesting we come to a group decision on that. @wics-uw/contributors do you have any thoughts?
2. We can pair on this, just get on IRC and I'll walk you through it.

Comment (meltedesco) [inline content/W2015/W15-event-research-showcase.md:4]: Personally, I prefer "academic" and it seems to be what people have actually been using so I vote we go for that.

Comment (fboxwala) [inline content/W2015/W15-event-research-showcase.md:4]: ++academic

Comment (evykassirer) [inline content/W2015/W15-event-research-showcase.md:4]: ++

**Comment by ehashman** [comment]
In the spirit of getting the other pull requests on the table resolved, I will make the necessary modifications and manually merge this since it's gone stale.

**Pull request merged**